netbox/data_netbox_json_circuits_circuit_terminations_list.go

```diff
@@ -56,7 +56,7 @@ func dataNetboxJSONCircuitsCircuitTerminationsListRead(ctx context.Context, d *s
 	params.Offset = &offset
 	for int64(len(tmp)) < desiredLength {
 		offset = int64(len(tmp))
-		if limit > desiredLength-offset {
```

Comment:
Why this change ?

Reply:
Fixed. I did run utils/generateJsonDatasources. That file is now changed to produce the same output as go fmt